target/src/package.json

```diff
@@ -1,10 +1,9 @@
 {
   "name": "antlr4ts",
-  "version": "0.4.0-dev",
+  "version": "0.4.1-alpha.0",
```

Comment:
Sorry for the late reply. The goal is to ensure that a user who checks out any arbitrary commit will get a build with exactly one of these characteristics:

1. The build result exactly matches one of our tagged releases, and has a version number matching that release
2. The build is from a commit which is not a tagged release, and has a version number ending in `-dev`